src/auth/feedback-secret.guard.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  Injectable,
+  CanActivate,
+  ExecutionContext,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { Env } from 'src/env';
+
+@Injectable()
+export class FeedbackSecretGuard implements CanActivate {
+  constructor() {}
```

Comment:
PullProofAI

Consider removing the constructor if it's empty as it doesn't serve any purpose.
```diff
-   constructor() {}
```



<!-- This is an auto-generated comment by PullProofAI -->